src/test/java/com/sampleweb/journey/CdVinylTests.java

```diff
@@ -0,0 +1,112 @@
+package com.sampleweb.journey;
+
+import com.sampleweb.BaseTest;
+import com.sampleweb.framework.PageNavigator;
+import com.sampleweb.pages.CdVinylPage;
+
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.Collection;
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+public class CdVinylTests  extends BaseTest {
+
+    private PageNavigator navigator = new PageNavigator();
+    private CdVinylPage cdVinylPage;
+
+    @BeforeMethod
+    public void setUp() throws Exception {
+        cdVinylPage = navigator.navigateToPage(driver, CdVinylPage.PATH, CdVinylPage.class);
+    }
+
+    @Test(groups = "regression")
+    public void navigateToPage() {
+        //CDs & Vinyl dropdown
+        String dropDownPath = "//select[@class=\"nav-search-dropdown searchSelect\"]/option[@value=\"search-alias=popular\"]";
+
+        driver.findElementByXPath(dropDownPath).click();
+
+        cdVinylPage.setSearchCriteria("").clickSubmitButton();
+
+        assertThat(cdVinylPage.isLoaded(), is(true));
+    }
+
+
+    @Test(groups = "regression")
+    public void titleTest() {
+        Assert.assertTrue( driver.getTitle().contains(cdVinylPage.TITLE) );
+    }
+
+    @Test(groups = "regression")
+    public void checkGlobalStoreSection() {
+        WebElement storeElement = driver.findElementByXPath(cdVinylPage.GLOBAL_STORE_CHECKBOX_PATH);
+
+        //if query can't find an object, fail test
```

Comment:
You don't actually need this check here since the method already throws an exception if it can't find any elements with the given xpath.